docs/changelog.md

```diff
@@ -7,6 +7,16 @@ follow [merged Pull request
 pages](https://github.com/dalibo/ldap2pg/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged).
 
 
+# Unreleased
+
+**Beware** when upgrading : **ldap2pg will drop roles having uppercase letter in
```

Comment:
@zurikus, what do you think of this ?

Reply:
I'm thinking of a way to rename.

Reply:
Got it. What do you think of this ?

Reply:
Yes, I think this is the right note for ldap2pg package users and it should be BOLD.
Truely saying double quotes support not only upper case letters but also "space" and other special characters as well.
http://joxi.ru/brR9x6QI70PGqA

Reply:
Right about spaces too, thanks for the hint.